kaguna/dimba-api
# New team creation controller.

#### What does this PR do?
- Enables the admin to add a new team.
#### Description of Task to be completed?
- Route to the creation controller.
- Create method to add new team.
- Integrate with circleci.
- Add tests.
#### How should this be manually tested?
- Run the server first `rails s`.
- Open `postman` and type `http://127.0.0.1:3000/teams/create_new_team` on the url tab.
- On the body section make sure to have the following keys with respective values. `name, description, location and nickname`
- Click send.
#### What are the relevant pivotal tracker stories? 
N/A
